### PR TITLE
Some builder docs improvements

### PR DESCRIPTION
**Pull request by thaJeztah**
found a "todo" I wrote down, to open a PR for the review suggestions I made in https://github.com/moby/moby/pull/29621, so here it is :smile:

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @vdemeester PTAL

File: docs/reference/builder.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ Practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-pr
 ## Usage
 
 The [`docker build`](commandline/build.md) command builds an image from
-a `Dockerfile` and a *context*. The build's context is the files at a specified
-location `PATH` or `URL`. The `PATH` is a directory on your local filesystem.
-The `URL` is a Git repository location.
+a `Dockerfile` and a *context*. The build's context is is the set of files at a
```
Comment (vdemeester) [inline docs/reference/builder.md]: `is is` 😛

File: docs/reference/commandline/build.md [hunk 2]
```diff
@@ -63,11 +63,11 @@ Options:
...
 
-Builds Docker images from a Dockerfile and a "context". A build's context is
-the files located in the specified `PATH` or `URL`. The build process can refer
-to any of the files in the context. For example, your build can use an
-[*ADD*](../builder.md#add) instruction to reference a file in the
-context.
+The `docker build` command builds Docker images from from a Dockerfile and a
+"context". A build's context is the set of files located in the specified
+`PATH` or `URL`. The build process can refer to any of the files in the
+context. For example, your build can use an [*ADD*](../builder.md#add)
```
Comment (vdemeester) [inline docs/reference/commandline/build.md]: should we use `COPY` here instead (*nit*)

**Comment by thaJeztah** [comment]
updated, PTAL 👍

File: docs/reference/commandline/build.md [hunk 3]
```diff
@@ -63,11 +63,11 @@ Options:
 
 ## Description
 
-Builds Docker images from a Dockerfile and a "context". A build's context is
-the files located in the specified `PATH` or `URL`. The build process can refer
-to any of the files in the context. For example, your build can use an
-[*ADD*](../builder.md#add) instruction to reference a file in the
-context.
+The `docker build` command builds Docker images from from a Dockerfile and a
```
Comment (dnephin) [inline docs/reference/commandline/build.md]: "from from"

Comment (vdemeester) [inline docs/reference/commandline/build.md]: ahah  @thaJeztah after *is is*, *from from* 😂 I think you're doing it to make sure we review it correctly 😝

Comment (thaJeztah) [inline docs/reference/commandline/build.md]: LOL! You're reviewing it wrong; this is a "high availability" from

**Comment by thaJeztah** [comment]
Changed `from` to a single-manager setup; PTAL

**Comment by thaJeztah** [comment]
Yay! And it's 💚. Pressing the button to make it 💜

**Pull request merged**